package.json

```diff
@@ -73,6 +73,7 @@
     "@types/jest": "^29.3.1",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
+    "@vitest/coverage-v8": "2.1.4",
```

Comment:
Does this require any further config?

Reply:
Added it to the vitest config in the package:
```
    coverage: {
      provider: 'v8',
    },
```